src/components/Icon.tsx

```diff
@@ -62,9 +62,10 @@ type IconName = keyof typeof ICONS;
 export interface IconProps {
   name: IconName;
   size?: number;
+  accessibilityLabel?: string;
 }
 
-export const Icon = ({name, size = 24}: IconProps) => {
+export const Icon = ({name, size = 24, accessibilityLabel}: IconProps) => {
   const IconImpl = ICONS[name];
-  return IconImpl ? <IconImpl width={size} height={size} /> : null;
+  return IconImpl ? <IconImpl accessible accessibilityLabel={accessibilityLabel} width={size} height={size} /> : null;
```

Comment:
Adding `accessible` to all icons creates an unnecessary tab-stop on decorative icons. Please remove.